### PR TITLE
Update Udop Scripts and Models

### PR DESCRIPTION
**Pull request by zinengtang**
- Update Udop Eval Scripts on DUE Benchmark
- Address Styles/Bugs
- Add Model Link

### CODE REVIEW

File: i-Code-Doc/benchmarker/__init__.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "config": UdopConfig,
+        "config2d": UdopConfig,
+        "tokenizer": UdopTokenizer,
+        "model_attr_name": "UdopUnimodel",
+        "wordgap_data_converter": None,
+        "pretraining": UdopUnimodelForConditionalGeneration,
+        "token_classification": UdopUnimodelForConditionalGeneration,
+    },
+    "UdopDual": {
+        "config": UdopConfig,
```
Comment (ziyi-yang) [inline i-Code-Doc/benchmarker/__init__.py:25]: seems like UDOP and UDOP dual are using the same config class?

File: i-Code-Doc/scripts/finetune_rvlcdip.sh [hunk 2]
```diff
@@ -1,8 +1,5 @@
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT license.
-
-CUDA_VISIBLE_DEVICES=0,1,2,3 PYTHONPATH='.' python run_rvlcdip.py \
---data_dir "./dataset/rvl-cdip" \
+CUDA_VISIBLE_DEVICES=0 PYTHONPATH='.' python run_rvlcdip.py \
```
Comment (ziyi-yang) [inline i-Code-Doc/scripts/finetune_rvlcdip.sh:1]: seems all scripts are changed to single-node finetuning. Have you tested on multi-GPU finetuning?

File: i-Code-Doc/core/models/udop_unimodel.py [hunk 3]
```diff
@@ -387,13 +387,19 @@ def forward(
...
+            # (self-attention position bias), (cross-attention weights), (cross-attention position bias)
+            #import pdb; pdb.set_trace()
             position_bias = layer_outputs[2]
             if self.is_decoder and encoder_hidden_states is not None:
                 encoder_decoder_position_bias = layer_outputs[4 if output_attentions else 3]
             # append next layer key value states
             if use_cache:
                 present_key_value_states = present_key_value_states + (present_key_value_state,)
 
+            #import pdb; pdb.set_trace()
```
Comment (ziyi-yang) [inline i-Code-Doc/core/models/udop_unimodel.py:401]: pls remove "import pdb; pdb.set_trace()" in all scripts.

File: i-Code-Doc/scripts/finetune_rvlcdip.sh (hunk 2 above)
Comment (zinengtang) [reply i-Code-Doc/scripts/finetune_rvlcdip.sh:1]: Yeah multi-GPU should work fine

File: i-Code-Doc/benchmarker/__init__.py (hunk 1 above)
Comment (zinengtang) [reply i-Code-Doc/benchmarker/__init__.py:25]: This is how the config is initially design I remembered. Because the parameters are mostly shared.

**Review by ziyi-yang — APPROVED** [review]

**Pull request merged**